definitions/json/CaseEvent/CaseEvent.json

```diff
@@ -1060,5 +1060,48 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "ShowSummary": "N"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "claimantTSE",
+    "Name": "Make an Application",
+    "Description": "Claimant Tell Something Else",
+    "PreConditionState(s)": "Submitted;Vetted;Rejected;Accepted;Closed",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "Y",
+    "EventEnablingCondition": "caseType=\"dummy\"",
+    "CallBackURLAboutToStartEvent": "${ET_COS_URL}/claimantTSE/aboutToStart",
+    "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/claimantTSE/aboutToSubmit",
+    "CallBackURLSubmittedEvent": "${ET_COS_URL}/claimantTSE/completeApplication",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "claimantTseAllApplications",
+    "Name": "All applications",
+    "Description": " ",
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "CallBackURLAboutToStartEvent": "${ET_COS_URL}/claimantTSE/displayTable",
+    "EndButtonLabel": "Close and return to case details",
+    "EventEnablingCondition": "caseType =\"dummy\""
+  },
```

Comment:
DiaplayOrder field may not be relevant as the event seems not enabled. Do we need to add it just to keep the formate/structure consistent?

Reply:
since this event is not shown in the drop down list, the order doesn’t matter and this doesn't affect the functionality